.github/workflows/main.yml

```diff
@@ -166,26 +170,46 @@ jobs:
         echo ::set-output name=TGZ_PACKAGE_PATH::$(ls build/*.tar.gz)
         echo ::set-output name=TGZ_PACKAGE_NAME::$(cd build && ls *.tar.gz)
 
+    # DEB Package
     - name: Upload the DEB package artifact (RelWithDebInfo only)
       if: matrix.build_type == 'RelWithDebInfo' && runner.os == 'Linux'
       uses: actions/upload-artifact@v2.3.1
       with:
         name: ${{ steps.package_locations.outputs.DEB_PACKAGE_NAME }}
         path: ${{ steps.package_locations.outputs.DEB_PACKAGE_PATH }}
 
+    - name: Release DEB package artifact (RelWithDebInfo only)
+      uses: softprops/action-gh-release@v0.1.14
+      if: matrix.build_type == 'RelWithDebInfo' && runner.os == 'Linux' && startsWith(github.ref, 'refs/tags/')
```

Comment:
I just want to check my understanding. This is only uploading a release if the current commit is tagged right?

Reply:
Yep! This step will only run when a tag is pushed